webapp/lib/firebase_constants.dart

```diff
@@ -1,13 +1,17 @@
 library firebase.constants;
 
+
 import 'dart:convert';
 import 'dart:html';
+import 'package:katikati_ui_lib/components/platform/platform_constants.dart' as platform_constants;
```

Comment:
this import (`platform_constants.dart`) should have fully replaced this file (`firebase_constants.dart`) (so `firebase_tools` would import from `kk_ui_lib` - can you add a TODO somewhere here that this should be done long term?

Reply:
Thanks @marianamarasoiu, good point